back/migrations/20220623130935-add-groups.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var dbm;
```

Comment:
Avoid using "var", prefer scoped instances like "const" or "let" unless you have a strong requirement